scripts/run-instrumented-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo =============================================
+echo Starting ADB Daemon
+echo =============================================
+adb start-server
+emulator @test -no-window -no-audio -wipe-data &
+sleep 30
+gradle -version
```

Comment:
Can we make this more prominent, like including it in the banner below with backticks?

Reply:
All of the commands or something specific?  I'm fine with making all of this more prominent. :-)

Reply:
It doesn't do a good job of this, but the Gradle version being run.